src/System.CommandLine/Resources.cs

```diff
@@ -8,11 +8,11 @@
 
 namespace System.CommandLine
 {
-    public class ValidationMessages
+    public class Resources
```

Comment:
I think the naming might be pointed in the wrong direction here. It's reflecting the implementation detail (that the underlying strings are resources) and not the use case (localizable messages). This is especially noticeable because it exposes parameterized methods, not strings.